cytominer_eval/tests/test_transform/test_transform.py

```diff
@@ -89,7 +89,7 @@ def test_metric_melt():
             similarity_metric="pearson",
             eval_metric="NOT SUPPORTED",
         )
-    assert "not supported. Available evaluation metrics:" in str(ve.value)
+    assert "NOT SUPPORTED not supported. Select one of" in str(ve.value)
```

Comment:
```suggestion
    assert "NOT SUPPORTED. Select one of" in str(ve.value)
```

Reply:
This will not pass the assertion - the error text is supposed to output what the user passed to `eval_metric`, which, in the case above, is "NOT SUPPORTED".

I agree that this is fairly confusing, so I'll change "NOT SUPPORTED" to "MISSING".